src/Module/Filer/SaveTag.php

```diff
@@ -52,13 +53,15 @@ public static function rawContent(array $parameters = [])
 		$logger->info('filer', ['tag' => $term, 'item' => $item_id]);
 
 		if ($item_id && strlen($term)) {
-			// file item
-			Model\FileTag::saveFile(local_user(), $item_id, $term);
+			$item = Model\Post::selectFirst(['uri-id'], ['id' => $item_id]);
+			if (!DBA::isResult($item)) {
+				return;				
```

Comment:
Shouldn't it throw a NotFound exception here?

Reply:
jupp